.github/ISSUE_TEMPLATE/rollout-issue.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Rollout issue
+about: 'Create issue for arcade-services rollout'
```

Comment:
```suggestion
about: Create issue for arcade-services rollout
```

Reply:
This was created by the GitHub templating engine itself, but we can give it a try and remove it to make it consistent with the other fields.